### PR TITLE
Support SDK-style projects with MSBuild SDK assets.

### PR DESCRIPTION
**Pull request by barnson**

### CODE REVIEW

File: src/WixToolset.MSBuild/Sdk/Sdk.props [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' ">$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)..\tools\wix.targets'))</WixTargetsPath>
+  </PropertyGroup>
```
Comment (rseanhall) [inline src/WixToolset.MSBuild/Sdk/Sdk.props:10]: Why is this necessary? Does build\WixToolset.MSBuild.props not get pulled in? If not, then we should probably import that instead of duplicating this here.

File: src/WixToolset.MSBuild/Sdk/Sdk.targets [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Import Project="$(WixTargetsPath)" Condition=" '$(WixTargetsPath)' != '' and Exists('$(WixTargetsPath)')" />
```
Comment (rseanhall) [inline src/WixToolset.MSBuild/Sdk/Sdk.targets:8]: Same thing here. I think we should have made a WixToolset.MSBuild.targets file that had this.

**Comment by rseanhall** [comment]
If we're going to stick with the single package approach, then the tests can probably be updated to import the Sdk.props and Sdk.targets to at least try to test what's in there even if they aren't SDK-style projects.

File: src/WixToolset.MSBuild/Sdk/Sdk.props (hunk 1 above)
Comment (barnson) [reply src/WixToolset.MSBuild/Sdk/Sdk.props:10]: Nope. But I can simplify it a bit with `<Import Project="..\build\WixToolset.MSBuild.props" />`
